flake.nix

```diff
@@ -5,7 +5,7 @@
     versions.url = "github:holochain/holochain?dir=versions/0_2";
     holonix.url = "github:holochain/holochain";
     holonix.inputs.versions.follows = "versions";
-    holonix.inputs.holochain.url = "github:holochain/holochain/holochain-0.2.2";
+    holonix.inputs.holochain.url = "github:holochain/holochain/holochain-0.2.3-beta-rc.1";
```

Comment:
we don't build this holochain tag on CI. see [here](https://github.com/holochain/holochain/blob/8521cee803aadb6e51fa967d57006eadf7861630/versions/0_2/flake.nix#L5) what's used for the 0_2 versions flake

Reply:
Right, thanks for clarifying @steveej

Reply:
The simple solution with that one is that we should get 0.2.3 released. Just waiting on https://github.com/holochain/holochain/pull/2951 and we can go ahead with that

Reply:
👍